src/firefly/js/api/ApiUtilImage.jsx

```diff
@@ -31,6 +31,8 @@ export {RangeValues} from '../visualize/RangeValues.js';
 export {WPConst, WebPlotRequest, findInvalidWPRKeys, confirmPlotRequest} from '../visualize/WebPlotRequest.js';
 export {RequestType} from '../visualize/RequestType';
 export {ExpandType, dispatchApiToolsView} from '../visualize/ImagePlotCntlr.js';
+export {CsysConverter} from '../visualize/CsysConverter.js';
+export {CCUtil} from '../visualize/CsysConverter.js';
```

Comment:
To use CCUtil, we need to add the following exports:
export {primePlot} from '../visualize/PlotViewUtil.js';
export {visRoot} from '../visualize/ImagePlotCntlr.js';